.github/workflows/sigstore-integration.yml

```
@@ -0,0 +1,78 @@
+      - name: Setup Sigstore container environment
+        id: sigstore-env
+        uses: sigstore/scaffolding/actions/setup-sigstore-env@8bd68672d418e5bd1b0ee1f2e2981874c3a30967 # v0.7.33
+
+      - name: Run container-based integration tests
+        working-directory: bindings/go/sigstore/integration_containers
+        env:
+          SIGSTORE_TRUSTED_ROOT: ${{ steps.sigstore-env.outputs.trusted-root }}
+          SIGSTORE_SIGNING_CONFIG: ${{ steps.sigstore-env.outputs.signing-config }}
+          SIGSTORE_OIDC_ISSUER: ${{ steps.sigstore-env.outputs.issuer-url }}
+          SIGSTORE_OIDC_IDENTITY: "foo@bar.com"
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2155:warning:2:8: Declare and assign separately to avoid masking return values

(shellcheck)